FateGrandAutomata/AndroidImpl.cs

```diff
@@ -55,6 +62,14 @@ public void MessageBox(string Title, string Message)
             });
         }
 
+        public void Highlight(Region Region, double Timeout)
+        {
+            HighlightView.AddRegion(Region);
+
+            Task.Delay(TimeSpan.FromSeconds(Timeout))
```

Comment:
This looks like an asynchronous call, won't the highlights disturb the screenshots then?

So I would add an `await` here.